site/content/docs/user/local-registry.md

```diff
@@ -22,6 +22,7 @@ with it enabled.
 set -o errexit
 
 # create registry container unless it already exists 
+CLUSTER_NAME="my-cluster"   # desired cluster name; default is "kind"
```

Comment:
maybe just make this `kind`?

Reply:
Can punt to a follow up

Reply:
Sorry you're much faster than I could respond:)
I think having a dummy name like that might draw the right attention that "it's your decision" -- especially with a note there; while naming it `kind` may cause someone think of "is that a keyword"...

Anyway, thanks for merging it!